src/main/java/com/gluonhq/substrate/target/IosTargetConfiguration.java

```diff
@@ -108,14 +108,21 @@ public IosTargetConfiguration(ProcessPaths paths, InternalProjectConfiguration c
 
     @Override
     List<String> getTargetSpecificAOTCompileFlags() throws IOException {
-
+        boolean graalvm22 = true;
```

Comment:
Maybe rename to `useInternalLlc`?

Reply:
what is internal?

Reply:
You added `internalLlcPath`, so I guess it makes sense to name the boolean like that

Reply:
those are 2 different things. We have a check on the internalLlcPath. If that is true, we know that the internal graalvm version is at least -ea+22. As a consequence, we can use the internally bundled llc.